admin_manual/installation/installation_wizard.rst

```diff
@@ -140,50 +140,210 @@ use :ref:`label-phpinfo` (Look for the **User/Group** line).
    ownership as above could result in some issues if the NFS mount is 
    lost.
 
-The easy way to set the correct permissions is to copy and run this script. 
-Replace the ``ocpath`` variable with the path to your Nextcloud directory, and 
-replace the ``htuser`` and ``htgroup`` variables with your HTTP user and group::
+The easy way to set the correct permissions is to copy and save this script to
+a suitable path location, perhaps ``/usr/local/bin/`` - then run the script
+from within your Nextcloud installation directory.  As per the comments in
+the script, you may setup additional environment variables on the command line
+if required.  The script can also be used to prepare for less strict
+permissions as required to do an upgrade::
 
  #!/bin/bash
- ocpath='/var/www/nextcloud'
- htuser='www-data'
- htgroup='www-data'
- rootuser='root'
-
- printf "Creating possible missing Directories\n"
- mkdir -p $ocpath/data
- mkdir -p $ocpath/assets
- mkdir -p $ocpath/updater
-
- printf "chmod Files and Directories\n"
- find ${ocpath} -type f -print0 | xargs -0 chmod 0640
- find ${ocpath} -type d -print0 | xargs -0 chmod 0750
-
- printf "chown Directories\n"
- chown -R ${rootuser}:${htgroup} ${ocpath}/
- chown -R ${htuser}:${htgroup} ${ocpath}/apps/
- chown -R ${htuser}:${htgroup} ${ocpath}/assets/
- chown -R ${htuser}:${htgroup} ${ocpath}/config/
- chown -R ${htuser}:${htgroup} ${ocpath}/data/
- chown -R ${htuser}:${htgroup} ${ocpath}/themes/
- chown -R ${htuser}:${htgroup} ${ocpath}/updater/
-
- chmod +x ${ocpath}/occ
-
- printf "chmod/chown .htaccess\n"
- if [ -f ${ocpath}/.htaccess ]
-  then
-   chmod 0644 ${ocpath}/.htaccess
-   chown ${rootuser}:${htgroup} ${ocpath}/.htaccess
- fi
- if [ -f ${ocpath}/data/.htaccess ]
-  then
-   chmod 0644 ${ocpath}/data/.htaccess
-   chown ${rootuser}:${htgroup} ${ocpath}/data/.htaccess
- fi
- 
+
+ # Script: set-nextcloud-perms.sh
+ #
+ # Usage:
+ #      /path/to/script/set-nextcloud-perms.sh [upgrade]
+ #
+ #  (run script whilst in your Nextcloud installation directory)
+ #
+ #
+ # Other Notes:
+ #
+ # If your "nc_data" path is special, then prefix the cmdline as
+ # with environment variables as required:
+ #   nc_data=/path/to/data
+ #
+ # If your distro or setup has non-standard htuser and/or htgroup,
+ # you should similarly prefix the environment variables as required:
+ #   htuser=special_user htgroup=special_group
+ #
+ # Example with a cmdline variable:
+ #   nc_data=/path/to/data /path/to/script/set-nextcloud-perms.sh [upgrade]
+
+ #
+ # Functions
+ #
+
+ do_cmd () {
+     echo -e "${@}\n\n$(${@})\n"
+     err_status=$?
+     [ ! ${err_status} -eq 0 ] && {
+         error_exit "Command failed with error: ${error_status}"
+     }
+ }
+
+ error_exit () {
+     echo -e "${1-"Unknown error"} -- exiting\n"
+     error_exit=${2:-1}
+     exit ${error_exit}
+ }
+
+
+ #
+ # Main
+ #
+
+ # Default DocumentRoot
+ DefaultDocumentRoot=/var/www/owncloud
```

Comment:
Okay, adjusted script.  Now doesn't use `DefaultDocumentRoot` variable, but it does check for occ in both `/var/www/nextcloud` then `/var/www/owncloud` locations to try and find `occ` file.